libraries/app/api.cpp

```diff
@@ -271,6 +271,37 @@ void network_broadcast_api::broadcast_block(const signed_block& b)
     }
 }
 
+class check_banned_operations_visitor
+{
+public:
+    using result_type = bool;
+
+    // available by default
+    template <typename Op> bool operator()(const Op&) const
+    {
+        return true;
+    }
+
+#ifdef LOCK_BLOGGING_API
```

Comment:
I prefer avoid define. There is no need to use it here and it makes makefile messy.